### ISSUE TITLE
could not find function "find_common_prefix"

### ISSUE THREAD

**Issue by christianrickert** [akoyabio/phenoptrReports#56]
One of our customers contacted us with a novel error message that indicates a regression from phenoptrReports v0.3.1 to v0.3.2 when using the "Analyze consolidated data" dialog. In particular, the error occurs when loading a consolidated data file:

![image](https://user-images.githubusercontent.com/19319377/148862210-ea83415d-0e58-49f9-a2b5-5ee05616bce2.png)

```R
Warning: Error in find_common_prefix: could not find function "find_common_prefix"
```
- R v4.1.1
- Rtools v4.0
- RStudio v1.4.1717
- phenoptr v0.3.1@9000
- phenoptrReports v0.3.2

The error does not occur with version 0.3.1 (head) of phenoptrReports.

Full R console output:

```R
R version 4.1.1 (2021-08-10) -- "Kick Things"
Copyright (C) 2021 The R Foundation for Statistical Computing
Platform: x86_64-w64-mingw32/x64 (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> phenoptrReports:::addin_30_analysis_app()
Loading required package: shiny
Warning: package ‘shiny’ was built under R version 4.1.2
Warning: package ‘tidyverse’ was built under R version 4.1.2
Warning: package ‘ggplot2’ was built under R version 4.1.2
Warning: package ‘tibble’ was built under R version 4.1.2
Warning: package ‘tidyr’ was built under R version 4.1.2
Warning: package ‘readr’ was built under R version 4.1.2
Warning: package ‘purrr’ was built under R version 4.1.2
Warning: package ‘dplyr’ was built under R version 4.1.2
Warning: package ‘stringr’ was built under R version 4.1.2
Warning: package ‘forcats’ was built under R version 4.1.2

Listening on http://127.0.0.1:5163
Rows: 5 Columns: 205                                                                                       
-- Column specification ------------------------------------------------
Delimiter: "\t"
chr  (13): tag, Path, Sample Name, Tissue Category, Process Region I...
dbl (192): Cell ID, Cell X Position, Cell Y Position, Category Regio...

i Use `spec()` to retrieve the full column specification for this data.
i Specify the column types or set `show_col_types = FALSE` to quiet this message.
Data is already in microns, no conversion performed                                                        
Warning: Error in find_common_prefix: could not find function "find_common_prefix"
  47: ifelse [C:\Program Files\R\R-4.1.1\library\phenoptrReports\analysis_app/server.R#71]
  46: ::
shiny
observe [C:\Program Files\R\R-4.1.1\library\phenoptrReports\analysis_app/server.R#71]
  45: <observer>
   2: shiny::runApp
   1: phenoptrReports:::addin_30_analysis_app
```

**Comment by AB-Kent** [comment]
Sorry about that! Fixed in new version 0.3.3. I suggest customer also update to the released version of phenoptr.

**Comment by christianrickert** [comment]
Thank you very much!
